docs/2/general/faq/security-faq.md

```diff
@@ -0,0 +1,105 @@
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+Is the Microshare application compliant with relevant data protection and privacy regulations, such as GDPR or SOC 2?
+{% endcapture %}
+
+{% capture content %}
+Microshare is SOC 2, IASME Cyber Essentials, and IASME Cyber Assurance (GDPR) complaint. We undergo annual third-party assessments as part of the recertification process to ensure we meet all data protection and privacy requirements. For more information, please visit the Microshare Data Security and Privacy Trust Center at https://www.microshare.io/microshares-trust-center-data-security/
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
+
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+Do you perform regular security audits and penetration testing of your software and systems?
+{% endcapture %}
+
+{% capture content %}
+Yes. Microshare performs both automated and manual vulnerability scans against our software and systems on at least a quarterly basis. In addition, we conduct third-party penetration testing on an annual basis. For more information, please see the Microshare Security Letter at https://www.microshare.io/wp-content/uploads/2022/08/Microshare-SOC-2-Type-2-Security-Letter.pdf 
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
+
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+Do you leverage encryption to protect data, both at rest and in transit?
+{% endcapture %}
+
+{% capture content %}
+Microshare utilizes encryption in transit and at‐rest by default. TLS 1.2/1.3 with SHA-256/AES-128 is used as a default encryption standard between servers and all clients with TLS 1.2/1.3 support. For more information, please see our Security Overview document at https://www.microshare.io/wp-content/uploads/2022/11/Security-1-pager-3.pdf
+{% endcapture %}
+
+{% include accordion.html title=title content=content key="security" %}
+
+
+<!-- ********************
+**    New Question     **
+********************* -->
+
+{% capture title %}
+What security measures do you implement during the software development process?
+{% endcapture %}
+
+{% capture content %}
+Microshare follows an iterative SDLC and is implementing the MS Security Development Lifecycle (MS SDL) as a security-specific overlay to our processes. OWASP standards are followed for the development and configuration for web accessible applications in APIs. All code and infrastructure changes are tracked in a source control repository. Any changes must be reviewed/approved before they are allowed to be submitted to the main branch.  For more information on our secure development process please see the Microshare Security Letter at  https://www.microshare.io/wp-content/uploads/2022/08/Microshare-SOC-2-Type-2-Security-Letter.pdf
```

Comment:
to put a link use "[]()" : [text appearing, could be the link](link)

Reply:
Addressed.